codice/ddf
# DDF-5511 Upgrade Hazelcast to 3.12.3

#### What does this PR do?
Upgrades Hazelcast to 3.12.3

#### Who is reviewing it? 

@codice/continuous-integration 
@mdang8 
@rfding 

#### Select relevant component teams: 

@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.
@brjeter
@clockard

#### How should this be tested?

This should be non-invasive and does not add any functionality to test. This branch can be checked out, and built using 'mvn clean install' to ensure that it passes all current compilation, unit test, and integration test steps

#### What are the relevant tickets?
Fixes: #5511 

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
